protocol/devops/README.md

```diff
@@ -0,0 +1,75 @@
+# DevOps Protocol
+
+A set of guidelines, checklists and tutorials for deploying web apps in a sane
+and reproduceable way.
+
+## Table of Contents
+
+* [Continuous Integration](#continuous-integration)
+* [Servers](#server-setup)
+* [Static websites](#static-websites)
+* [DNS](#server-setup)
+* [SSL](#server-setup)
+    * [Sabayon Samples](#sabayon-samples)
+
+## Continuous Integration
+
+* Use [SemaphoreCI] ([TravisCI] is also fine for open source, but tends to be
+    slower)
+* Set up automatic deploys to staging (from the `master` branch), and manual
+    deploys to production
+* Ensure the CI server is capable of deploying to all environments, so that no
```

Comment:
Yes, `heroku pipelines:promote` was what I meant.

Well, I see how that could be true, but we do have some clients using Heroku for production environments. Whether you should prefer using the CI or Heroku Pipelines I'm not able to tell, so far as I have experienced they're pretty equivalent. I just like Heroku better because of the CLI.